src/docker.rs

```diff
@@ -311,11 +315,14 @@ pub fn run(
         docker.args(&["-e", &format!("CROSS_DEBUG={value}")]);
     }
 
-    if let Ok(value) = env::var("DOCKER_OPTS") {
-        let opts = shell_words::split(&value)
-            .wrap_err_with(|| format!("could not parse docker opts of {}", value))?;
-        docker.args(&opts);
-    }
+    if let Ok(value) = env::var("CROSS_CONTAINER_OPTS") {
+        if env::var("DOCKER_OPTS").is_ok() {
+            eprintln!("Warning: using both `CROSS_CONTAINER_OPTS` and `DOCKER_OPTS`.");
+        }
+        docker.args(&parse_docker_opts(&value)?);
+    } else if let Ok(value) = env::var("DOCKER_OPTS") {
```

Comment:
We could warn users about the old environment variable and suggest using the new one. So we could eventually remove or error out.

Reply:
no need to error out, think it's fine to silently remove it after a few releases

Reply:
It'd be nice to add a comment in code so we can easily find it in future.

Reply:
I've added a FIXME which will let us know to remove it upon deprecating `DOCKER_OPTS`.

https://github.com/cross-rs/cross/pull/772/files#diff-4cf895b046f53b02266b843eb9440dec88b857e039a624a5310502e3dd7eb858R304-R306